### PR TITLE
Added support for NFS server in Soperator

### PR DESCRIPTION
**Pull request by d3vil-st**

### CODE REVIEW

**Comment by dstaroff** [comment]
Hi @d3vil-st 
At first glance this PR looks valid. We need some time to take a closer look

File: soperator/installations/example/main.tf [hunk 1]
```diff
@@ -68,6 +68,24 @@ module "filestore" {
...
+  source         = "../../../modules/nfs-server"
+  parent_id      = data.nebius_iam_v1_project.this.id
+  subnet_id      = data.nebius_vpc_v1_subnet.this.id
+  ssh_user_name  = "soperator"
+  ssh_public_key = var.slurm_login_ssh_root_public_keys[0]
+  nfs_ip_range   = data.nebius_vpc_v1_subnet.this.ipv4_private_pools.pools[0].cidrs[0].cidr
+  nfs_size       = var.nfs.size_gibibytes * 1024 * 1024 * 1024
+  nfs_path       = "/mnt/nfs"
+  platform       = "cpu-e2"
+  preset         = "16vcpu-64gb"
```
Comment (rdjjke) [inline soperator/installations/example/main.tf:82]: We should also make the preset configurable.

Comment (d3vil-st) [reply soperator/installations/example/main.tf]: Done

**Review by malibora — APPROVED** [review]

**Comment by dstaroff** [comment]
Guys, AFAIR on latest Slurm-CSA sync we decided that it's kinda raw solution and lacks some configuration. At least, we have to support configuration of mount path. @d3vil-st can you elaborate on this?

**Review by dstaroff — APPROVED** [review]

**Pull request merged**